### ISSUE TITLE
Unable to access Accessibility menu on keyboard type of Ctrl  + U

### ISSUE THREAD

**Issue by ab668685** [topcoder-platform/qa-fun#163]
Steps : -
1.  Hit URL -  https://www.topcoder.com/
2. Login with topcoder credentials.
3. Enter Ctrl + U

Expected Result :Should be able to access  Accessibility menu
Actual Result : Unable to access Accessibility menu until its been manually selected

Browser : Chrome 80

![image](https://user-images.githubusercontent.com/61318135/78262103-bd616700-751d-11ea-8701-31117c1b430b.png)

**Comment by winterflame** [comment]
not UI bug

**Comment by winterflame** [comment]
@ab668685 "Ctrl + U" is used for opening/closing the menu, which is a functionality (not UI). It's working well.
